### PR TITLE
Fix RSpec tests and code indent

### PR DESCRIPTION
**Pull request by shhavel**
Fixed RSpec tests
Set code indent to 2 spaces
Updated code spinets in README to highlight code 

Travis build - https://travis-ci.org/shhavel/evercam-ruby
Thanks

### CODE REVIEW

**Comment by marcoherbst** [comment]
Many thanks @shhavel , that's helpful.

The core developers are on holidays for the rest of this week, so it might be a few days until they get to merge this. 

Nice one :)

File: lib/evercam/api.rb [hunk 1]
```diff
@@ -168,40 +168,40 @@ def handle_raw(response)
...
+      url.string
     end
 
     # This method generates a versioned path for an API endpoint.
     #
     # ==== Parameters
     # suffix::  The API call dependent part of the path.
     def api_path(suffix)
-       ((/.+\..+$/ =~ suffix) != 0) ? "/v1#{suffix}" : "/v1#{suffix}"
+      "/v1#{suffix}"
```
Comment (ijunaidfarooq) [inline lib/evercam/api.rb]: any reasons you removed condition?

File: lib/evercam/api.rb [hunk 2]
```diff
@@ -211,10 +211,10 @@ def endpoint_url(suffix)
...
     # status::  The HTTP status code associated with the response.
     def handle_error_response(data, status)
-       message = (data["message"] || "Evercam API call returned an error.")
-       @logger.error "API response contains error details.\nMessage: #{message}\n"\
-                     "Code: #{data["code"]}\nStatus: #{status}"
-       raise EvercamError.new(message, data["code"], status, *data["context"])
+      message = (data["message"] || "Evercam API call returned an error.")
+      @logger.error "API response contains error details.\nMessage: #{message}\n"\
+        "Code: #{data["code"]}\nStatus: #{status}"
+        raise EvercamError.new(message, data["code"], status, *data["context"])
```
Comment (ijunaidfarooq) [inline lib/evercam/api.rb]: This indentation is not needed here.

File: lib/evercam/api.rb (hunk 1 above)
Comment (shhavel) [inline lib/evercam/api.rb]: I think in both cases it returns the same result.

File: lib/evercam/api.rb (hunk 2 above)
Comment (shhavel) [inline lib/evercam/api.rb]: Yes, will fix it, thanks.

**Pull request closed without merging**